presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -3128,6 +3129,19 @@ public void testIn()
         assertQuery("SELECT orderkey FROM orders WHERE orderkey IN (1, 2, 3)");
         assertQuery("SELECT orderkey FROM orders WHERE orderkey IN (1.5, 2.3)");
         assertQuery("SELECT orderkey FROM orders WHERE totalprice IN (1, 2, 3)");
+        assertQuery("SELECT x FROM (values 3, 100) t(x) WHERE x IN (2147483649)", "SELECT * WHERE false");
+        assertQuery("SELECT x FROM (values 3, 100) t(x) WHERE x IN (3, cast(rand() * 12345678910 as bigint))", "values 3");
```

Comment:
Remove this line. This is covered by the tests below, and it's potentially flaky (rand happens to return 100).